draft-thomson-http-oblivious.md

```diff
@@ -355,7 +355,7 @@ follows:
 In pseudocode, this procedure is as follows:
 
 ~~~
-secret = context.Export("secret", Nk)
+secret = context.Export("request", Nk)
```

Comment:
For HPKE we received a lot of feedback that it was most descriptive when the label matched the variable name, so how about:

```suggestion
response_secret = context.Export("response secret", Nk)
```

Or something to that effect?

Reply:
I'm happy with the simple symmetry of using "request" to construct a request and "response" to construct a response.  Qualifying with " secret" implies that there are other things we might need for responses.

This is a simple enough process to follow that we probably don't need to be more explicit.

Reply:
👍 I definitely don't feel strongly about this, other than that this label should (I think) be "response" since it's used to generate response encryption keys.